Fixes following error building glibc-2.2.5 with gcc-3.4:
td_init.c: In function `td_init':
td_init.c:30: error: parse error before string constant
td_init.c:30: error: parse error before string constant
make[2]: *** [/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.2.5/build-glibc/linuxthreads_db/td_init.os] Error 1

# Retrieved by diffing a +-1 day rng around the time mentioned
# in http://sources.redhat.com/ml/glibc-cvs/2001-q4/msg00654.html
# cvs -z 9 -d :pserver:anoncvs@sources.redhat.com:/cvs/glibc diff -u -D"2001-12-27 08:41:30" -D"2001-12-29 08:41:32"
# then fixing paths.

Index: ChangeLog
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/ChangeLog,v
retrieving revision 1.41
retrieving revision 1.42
diff -u -r1.41 -r1.42
--- libc/linuxthreads_db/ChangeLog	13 Apr 2001 00:09:54 -0000	1.41
+++ libc/linuxthreads_db/ChangeLog	29 Dec 2001 00:59:58 -0000	1.42
@@ -1,3 +1,43 @@
+2001-12-28  Andreas Jaeger  <aj@suse.de>
+
+	* td_init.c (td_init): Don't use __FUNCTION__ as literal.
+	* td_log.c (td_log): Likewise.
+	* td_ta_delete.c (td_ta_delete): Likewise.
+	* td_ta_get_nthreads.c (td_ta_get_nthreads): Likewise.
+	* td_ta_get_ph.c (td_ta_get_ph): Likewise.
+	* td_ta_map_id2thr.c (td_ta_map_id2thr): Likewise.
+	* td_ta_map_lwp2thr.c (td_ta_map_lwp2thr): Likewise.
+	* td_ta_new.c (td_ta_new): Likewise.
+	* td_ta_clear_event.c (td_ta_clear_event): Likewise.
+	* td_ta_enable_stats.c (td_ta_enable_stats): Likewise.
+	* td_ta_event_addr.c (td_ta_event_addr): Likewise.
+	* td_ta_event_getmsg.c (td_ta_event_getmsg): Likewise.
+	* td_ta_get_stats.c (td_ta_get_stats): Likewise.
+	* td_ta_reset_stats.c (td_ta_reset_stats): Likewise.
+	* td_ta_set_event.c (td_ta_set_event): Likewise.
+	* td_ta_setconcurrency.c (td_ta_setconcurrency): Likewise.
+	* td_ta_thr_iter.c (td_ta_thr_iter): Likewise.
+	* td_ta_tsd_iter.c (td_ta_tsd_iter): Likewise.
+	* td_thr_clear_event.c (td_thr_clear_event): Likewise.
+	* td_thr_dbresume.c (td_thr_dbresume): Likewise.
+	* td_thr_dbsuspend.c (td_thr_dbsuspend): Likewise.
+	* td_thr_event_enable.c (td_thr_event_enable): Likewise.
+	* td_thr_event_getmsg.c (td_thr_event_getmsg): Likewise.
+	* td_thr_get_info.c (td_thr_get_info): Likewise.
+	* td_thr_getfpregs.c (td_thr_getfpregs): Likewise.
+	* td_thr_getgregs.c (td_thr_getgregs): Likewise.
+	* td_thr_getxregs.c (td_thr_getxregs): Likewise.
+	* td_thr_getxregsize.c (td_thr_getxregsize): Likewise.
+	* td_thr_set_event.c (td_thr_set_event): Likewise.
+	* td_thr_setfpregs.c (td_thr_setfpregs): Likewise.
+	* td_thr_setgregs.c (td_thr_setgregs): Likewise.
+	* td_thr_setprio.c (td_thr_setprio): Likewise.
+	* td_thr_setsigpending.c (td_thr_setsigpending): Likewise.
+	* td_thr_setxregs.c (td_thr_setxregs): Likewise.
+	* td_thr_sigsetmask.c (td_thr_sigsetmask): Likewise.
+	* td_thr_tsd.c (td_thr_tsd): Likewise.
+	* td_thr_validate.c (td_thr_validate): Likewise.
+
 2001-04-12  Ulrich Drepper  <drepper@redhat.com>
 
 	* td_ta_map_id2thr.c: If thread terminated return TD_NOTHR.
Index: td_init.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_init.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_init.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_init.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Initialization function of thread debugger support library.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -27,6 +27,6 @@
 td_init (void)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_init");
   return TD_OK;
 }
Index: td_log.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_log.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_log.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_log.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Noop, left for historical reasons.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,8 +25,8 @@
 td_log (void)
 {
   /* This interface is deprecated in the Sun interface.  We provide it
-     for compatibility but don't do anyhting ourself.  We might in
+     for compatibility but don't do anything ourself.  We might in
      future do some logging if this seems reasonable.  */
-  LOG (__FUNCTION__);
+  LOG ("td_log");
   return TD_OK;
 }
Index: td_ta_clear_event.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_clear_event.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_clear_event.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_clear_event.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Globally disable events.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -29,7 +29,7 @@
   td_thr_events_t old_event;
   int i;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_clear_event");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_delete.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_delete.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_delete.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_delete.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Detach to target process.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -26,7 +26,7 @@
 td_err_e
 td_ta_delete (td_thragent_t *ta)
 {
-  LOG (__FUNCTION__);
+  LOG ("td_ta_delete");
 
   /* Safety check.  */
   if (ta == NULL || __td_agent_list == NULL)
Index: td_ta_enable_stats.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_enable_stats.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_enable_stats.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_enable_stats.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Enable collection of statistics for process.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,7 +25,7 @@
 td_ta_enable_stats (const td_thragent_t *ta, int enable)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_ta_enable_stats");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_event_addr.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_event_addr.c,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -r1.5 -r1.6
--- libc/linuxthreads_db/td_ta_event_addr.c	6 Jul 2001 05:27:23 -0000	1.5
+++ libc/linuxthreads_db/td_ta_event_addr.c	28 Dec 2001 16:41:29 -0000	1.6
@@ -27,7 +27,7 @@
   td_err_e res = TD_NOEVENT;
   int idx = -1;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_event_addr");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_event_getmsg.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_event_getmsg.c,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -r1.5 -r1.6
--- libc/linuxthreads_db/td_ta_event_getmsg.c	6 Jul 2001 05:27:23 -0000	1.5
+++ libc/linuxthreads_db/td_ta_event_getmsg.c	28 Dec 2001 16:41:29 -0000	1.6
@@ -1,5 +1,5 @@
 /* Retrieve event.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -32,7 +32,7 @@
   td_eventbuf_t event;
   psaddr_t addr;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_event_getmsg");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_get_nthreads.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_get_nthreads.c,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -r1.5 -r1.6
--- libc/linuxthreads_db/td_ta_get_nthreads.c	6 Jul 2001 05:27:23 -0000	1.5
+++ libc/linuxthreads_db/td_ta_get_nthreads.c	28 Dec 2001 16:41:29 -0000	1.6
@@ -25,7 +25,7 @@
 {
   psaddr_t addr;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_get_nthreads");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_get_ph.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_get_ph.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_get_ph.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_get_ph.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Get external process handle.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -24,7 +24,7 @@
 td_err_e
 td_ta_get_ph (const td_thragent_t *ta, struct ps_prochandle **ph)
 {
-  LOG (__FUNCTION__);
+  LOG ("td_ta_get_ph");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_get_stats.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_get_stats.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_get_stats.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_get_stats.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Retrieve statistics for process.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,7 +25,7 @@
 td_ta_get_stats (const td_thragent_t *ta, td_ta_stats_t *statsp)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_ta_get_stats");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_map_id2thr.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_map_id2thr.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_ta_map_id2thr.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_ta_map_id2thr.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -28,7 +28,7 @@
   struct _pthread_descr_struct pds;
   int pthread_threads_max;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_map_id2thr");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_map_lwp2thr.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_map_lwp2thr.c,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -r1.5 -r1.6
--- libc/linuxthreads_db/td_ta_map_lwp2thr.c	6 Jul 2001 05:27:23 -0000	1.5
+++ libc/linuxthreads_db/td_ta_map_lwp2thr.c	28 Dec 2001 16:41:29 -0000	1.6
@@ -1,5 +1,5 @@
 /* Which thread is running on an lwp?
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -34,7 +34,7 @@
 # define num 1
 #endif
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_map_lwp2thr");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_new.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_new.c,v
retrieving revision 1.10
retrieving revision 1.11
diff -u -r1.10 -r1.11
--- libc/linuxthreads_db/td_ta_new.c	6 Jul 2001 05:27:23 -0000	1.10
+++ libc/linuxthreads_db/td_ta_new.c	28 Dec 2001 16:41:29 -0000	1.11
@@ -35,7 +35,7 @@
   psaddr_t addr;
   struct agent_list *elemp;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_new");
 
   /* Get the global event mask.  This is one of the variables which
      are new in the thread library to enable debugging.  If it is
Index: td_ta_reset_stats.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_reset_stats.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_reset_stats.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_reset_stats.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Reset statistics.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,7 +25,7 @@
 td_ta_reset_stats (const td_thragent_t *ta)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_ta_reset_stats");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_set_event.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_set_event.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_ta_set_event.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_ta_set_event.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -1,5 +1,5 @@
 /* Globally enable events.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -29,7 +29,7 @@
   td_thr_events_t old_event;
   int i;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_set_event");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_setconcurrency.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_setconcurrency.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_ta_setconcurrency.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_ta_setconcurrency.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Set suggested concurrency level for process.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,7 +25,7 @@
 td_ta_setconcurrency (const td_thragent_t *ta, int level)
 {
   /* This is something LinuxThreads does not support.  */
-  LOG (__FUNCTION__);
+  LOG ("td_ta_setconcurrency");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_thr_iter.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_thr_iter.c,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -r1.11 -r1.12
--- libc/linuxthreads_db/td_ta_thr_iter.c	6 Jul 2001 05:27:23 -0000	1.11
+++ libc/linuxthreads_db/td_ta_thr_iter.c	28 Dec 2001 16:41:29 -0000	1.12
@@ -1,5 +1,5 @@
 /* Iterate over a process's threads.
-   Copyright (C) 1999, 2000 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2000, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -86,7 +86,7 @@
 # define num 1
 #endif
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_thr_iter");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_ta_tsd_iter.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_ta_tsd_iter.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_ta_tsd_iter.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_ta_tsd_iter.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -1,5 +1,5 @@
 /* Iterate over a process's thread-specific data.
-   Copyright (C) 1999, 2000 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2000, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -29,7 +29,7 @@
   int pthread_keys_max;
   int cnt;
 
-  LOG (__FUNCTION__);
+  LOG ("td_ta_tsd_iter");
 
   /* Test whether the TA parameter is ok.  */
   if (! ta_ok (ta))
Index: td_thr_clear_event.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_clear_event.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_thr_clear_event.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_thr_clear_event.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Disable specific event for thread.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -31,7 +31,7 @@
   td_thr_events_t old_event;
   int i;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_clear_event");
 
   /* Write the new value into the thread data structure.  */
   if (ps_pdread (th->th_ta_p->ph,
Index: td_thr_dbresume.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_dbresume.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_dbresume.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_dbresume.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Resume execution of given thread.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_dbresume (const td_thrhandle_t *th)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_dbresume");
   return TD_NOCAPAB;
 }
Index: td_thr_dbsuspend.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_dbsuspend.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_dbsuspend.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_dbsuspend.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Suspend execution of given thread.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_dbsuspend (const td_thrhandle_t *th)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_dbsuspend");
   return TD_NOCAPAB;
 }
Index: td_thr_event_enable.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_event_enable.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_thr_event_enable.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_thr_event_enable.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Enable event process-wide.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -28,7 +28,7 @@
      const td_thrhandle_t *th;
      int onoff;
 {
-  LOG (__FUNCTION__);
+  LOG ("td_thr_event_enable");
 
   /* Write the new value into the thread data structure.  */
   if (ps_pdwrite (th->th_ta_p->ph,
Index: td_thr_event_getmsg.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_event_getmsg.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/linuxthreads_db/td_thr_event_getmsg.c	6 Jul 2001 05:27:23 -0000	1.3
+++ libc/linuxthreads_db/td_thr_event_getmsg.c	28 Dec 2001 16:41:29 -0000	1.4
@@ -1,5 +1,5 @@
 /* Retrieve event.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -29,7 +29,7 @@
 {
   td_eventbuf_t event;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_event_getmsg");
 
   /* Read the even structure from the target.  */
   if (ps_pdread (th->th_ta_p->ph,
Index: td_thr_get_info.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_get_info.c,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -r1.9 -r1.10
--- libc/linuxthreads_db/td_thr_get_info.c	6 Jul 2001 05:27:23 -0000	1.9
+++ libc/linuxthreads_db/td_thr_get_info.c	28 Dec 2001 16:41:29 -0000	1.10
@@ -1,5 +1,5 @@
 /* Get thread information.
-   Copyright (C) 1999, 2000 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2000, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -29,7 +29,7 @@
 {
   struct _pthread_descr_struct pds;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_get_info");
 
   /* Get the thread descriptor.  */
   if (ps_pdread (th->th_ta_p->ph, th->th_unique, &pds,
Index: td_thr_getfpregs.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_getfpregs.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_thr_getfpregs.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_thr_getfpregs.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -26,7 +26,7 @@
 {
   struct _pthread_descr_struct pds;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_getfpregs");
 
   /* We have to get the state and the PID for this thread.  */
   if (ps_pdread (th->th_ta_p->ph, th->th_unique, &pds,
Index: td_thr_getgregs.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_getgregs.c,v
retrieving revision 1.8
retrieving revision 1.9
diff -u -r1.8 -r1.9
--- libc/linuxthreads_db/td_thr_getgregs.c	6 Jul 2001 05:27:23 -0000	1.8
+++ libc/linuxthreads_db/td_thr_getgregs.c	28 Dec 2001 16:41:29 -0000	1.9
@@ -26,7 +26,7 @@
 {
   struct _pthread_descr_struct pds;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_getgregs");
 
   /* We have to get the state and the PID for this thread.  */
   if (ps_pdread (th->th_ta_p->ph, th->th_unique, &pds,
Index: td_thr_getxregs.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_getxregs.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_getxregs.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_getxregs.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Get a thread's extra state register set.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_getxregs (const td_thrhandle_t *th, void *xregs)
 {
   /* XXX This might be platform specific.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_getxregs");
   return TD_NOXREGS;
 }
Index: td_thr_getxregsize.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_getxregsize.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_getxregsize.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_getxregsize.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Get the size of the extra state register set for this architecture.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_getxregsize (const td_thrhandle_t *th, int *sizep)
 {
   /* XXX This might be platform specific.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_getxregsize");
   return TD_NOXREGS;
 }
Index: td_thr_set_event.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_set_event.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_thr_set_event.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_thr_set_event.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -1,5 +1,5 @@
 /* Enable specific event for thread.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -31,7 +31,7 @@
   td_thr_events_t old_event;
   int i;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_set_event");
 
   /* Write the new value into the thread data structure.  */
   if (ps_pdread (th->th_ta_p->ph,
Index: td_thr_setfpregs.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_setfpregs.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_thr_setfpregs.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_thr_setfpregs.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -26,7 +26,7 @@
 {
   struct _pthread_descr_struct pds;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_setfpregs");
 
   /* We have to get the state and the PID for this thread.  */
   if (ps_pdread (th->th_ta_p->ph, th->th_unique, &pds,
Index: td_thr_setgregs.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_setgregs.c,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- libc/linuxthreads_db/td_thr_setgregs.c	6 Jul 2001 05:27:23 -0000	1.6
+++ libc/linuxthreads_db/td_thr_setgregs.c	28 Dec 2001 16:41:29 -0000	1.7
@@ -26,7 +26,7 @@
 {
   struct _pthread_descr_struct pds;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_setgregs");
 
   /* We have to get the state and the PID for this thread.  */
   if (ps_pdread (th->th_ta_p->ph, th->th_unique, &pds,
Index: td_thr_setprio.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_setprio.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_setprio.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_setprio.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Set a thread's priority.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_setprio (const td_thrhandle_t *th, int prio)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_setprio");
   return TD_OK;
 }
Index: td_thr_setsigpending.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_setsigpending.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_setsigpending.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_setsigpending.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Raise a signal for a thread.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -26,6 +26,6 @@
 		      const sigset_t *ss)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_setsigpending");
   return TD_OK;
 }
Index: td_thr_setxregs.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_setxregs.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_setxregs.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_setxregs.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Set a thread's extra state register set.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_setxregs (const td_thrhandle_t *ta, const void *addr)
 {
   /* XXX This might have to be platform specific.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_setxregs");
   return TD_NOXREGS;
 }
Index: td_thr_sigsetmask.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_sigsetmask.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_sigsetmask.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_sigsetmask.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Set a thread's signal mask.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -25,6 +25,6 @@
 td_thr_sigsetmask (const td_thrhandle_t *th, const sigset_t *ss)
 {
   /* XXX We have to figure out what has to be done.  */
-  LOG (__FUNCTION__);
+  LOG ("td_thr_sigsetmask");
   return TD_OK;
 }
Index: td_thr_tsd.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_tsd.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/linuxthreads_db/td_thr_tsd.c	6 Jul 2001 05:27:23 -0000	1.2
+++ libc/linuxthreads_db/td_thr_tsd.c	28 Dec 2001 16:41:29 -0000	1.3
@@ -1,5 +1,5 @@
 /* Get a thread-specific data pointer for a thread.
-   Copyright (C) 1999 Free Software Foundation, Inc.
+   Copyright (C) 1999, 2001 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@cygnus.com>, 1999.
 
@@ -33,7 +33,7 @@
   unsigned int idx2nd;
   void *p;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_tsd");
 
   /* Get the thread descriptor.  */
   if (ps_pdread (th->th_ta_p->ph, th->th_unique, &pds,
Index: td_thr_validate.c
===================================================================
RCS file: /cvs/glibc/libc/linuxthreads_db/td_thr_validate.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/linuxthreads_db/td_thr_validate.c	6 Jul 2001 05:27:23 -0000	1.4
+++ libc/linuxthreads_db/td_thr_validate.c	28 Dec 2001 16:41:29 -0000	1.5
@@ -28,7 +28,7 @@
   int pthread_threads_max = th->th_ta_p->pthread_threads_max;
   int cnt;
 
-  LOG (__FUNCTION__);
+  LOG ("td_thr_validate");
 
   /* Now get all descriptors, one after the other.  */
   for (cnt = 0; cnt < pthread_threads_max; ++cnt, ++handles)
